### DISCUSSION TITLE
Help needed with RNode LoRa Interface

### DISCUSSION THREAD

**Discussion by Fremen2001** [markqvist/Reticulum#235] [Help & Questions]
Hello,
I'm having two issues with rns and rnode. I've installed rnode on a Lilygo LoRa v2 and everything is ok, but when I run rnodeconf --info,
if I'm not root it fail the signature check. Not sure if this is an actual issue or not.
The real issue is when I try rnstatus after adding the LoRa interface in ./reticulum/config I always get and error
I tried every possible solution with no luck, this is the debug output:

`python3 -m pdb .local/lib/python3.9/site-packages/RNS/Utilities/rnstatus.py
> /home/umbrella/.local/lib/python3.9/site-packages/RNS/Interfaces/RNodeInterface.py(168)__init__()
-> if (self.frequency < RNodeInterface.FREQ_MIN or self.frequency > RNodeInterface.FREQ_MAX):
(Pdb)
TypeError: '<' not supported between instances of 'NoneType' and 'int'
> /home/umbrella/.local/lib/python3.9/site-packages/RNS/Interfaces/RNodeInterface.py(168)__init__()
-> if (self.frequency < RNodeInterface.FREQ_MIN or self.frequency > RNodeInterface.FREQ_MAX):
(Pdb)
--Return--
> /home/umbrella/.local/lib/python3.9/site-packages/RNS/Interfaces/RNodeInterface.py(168)__init__()->None
-> if (self.frequency < RNodeInterface.FREQ_MIN or self.frequency > RNodeInterface.FREQ_MAX):
(Pdb)
TypeError: '<' not supported between instances of 'NoneType' and 'int'
> /home/umbrella/.local/lib/python3.9/site-packages/RNS/Reticulum.py(841)__apply_config()
-> interface = RNodeInterface.RNodeInterface(
(Pdb)
> /home/umbrella/.local/lib/python3.9/site-packages/RNS/Reticulum.py(903)__apply_config()
-> except Exception as e:
(Pdb)
> /home/umbrella/.local/lib/python3.9/site-packages/RNS/Reticulum.py(904)__apply_config()
-> RNS.log("The interface \""+name+"\" could not be created. Check your configuration file for errors!", RNS.LOG_ERROR)
(Pdb)
`
Any suggestion is really appreciated. BTW I love this project :)

Fremen

**Comment by markqvist** [comment]
This may be a bug. I am looking into it.

One thing though, make sure your permissions are not weird on `~/.config/rnodeconf`! All files in there must be readable and writable by the user account you run `rnodeconf` as.

If not ...

```
chown -R youruser:youruser ~/.config/rnodeconf
chmod -R u+rw ~/.config/rnodeconf
```

... will usually do the trick.

**Reply by Fremen2001** [reply]
thanks for the swift reply!
you are right, I forgot to add my user to the dialout group so I did in fact run rnodeconf as root and all the files ended up under /root, that's the reason of the failed signature check :)
about the other issue, I'm running ParrotOS (debian based) python 3.9.2. just in case.
Thanks!
